pallets/runtime/develop/src/runtime.rs

```diff
@@ -529,6 +534,16 @@ impl pallet_utility::Trait for Runtime {
     type Call = Call;
 }
 
+impl EnactProposalMaker<Origin, Call> for Runtime {
+    fn is_pip_id_valid(id: PipId) -> bool {
+        Pips::is_proposal_id_valid(id)
+    }
+
+    fn enact_referendum_call(id: PipId) -> Call {
+        Call::Pips(pallet_pips::Call::enact_referendum(id))
+    }
```

Comment:
isn't it good to have the definition in the PIP module instead of here?

Reply:
That is impossible because the first `Call::Pips` is actually an `enum` defined in `runtime.rs`.
The `EnactProposalMaker` trait is just an utility to resolve the **circle dependency** between modules:

-  the **Runtime module**, where `Call` is defined, 
-  the  **Pips Module**, where `enact_referendum` fn and `is_proposal_id_valid` fn are defined,
-  and the **Committee Module**, where this helper trait is used.